assets/js/src/common/confirm_alert.jsx

```diff
@@ -1,65 +1,42 @@
 import MailPoet from 'mailpoet';
 import PropTypes from 'prop-types';
 import React from 'react';
-import ReactDOM from 'react-dom';
 
-class ConfirmAlert extends React.Component {
-  static propTypes = {
-    title: PropTypes.string,
-    message: PropTypes.string.isRequired,
-    cancelLabel: PropTypes.string,
-    confirmLabel: PropTypes.string,
-    onConfirm: PropTypes.func.isRequired,
-  }
-
-  static defaultProps = {
-    title: MailPoet.I18n.t('confirmTitle'),
-    cancelLabel: MailPoet.I18n.t('cancelLabel'),
-    confirmLabel: MailPoet.I18n.t('confirmLabel'),
-  }
-
-  close = () => {
-    const modalElement = document.getElementById('mailpoet_confirm_alert');
-    ReactDOM.unmountComponentAtNode(modalElement);
-    modalElement.className += ' mailpoet_modal_overlay_hidden';
-  }
-
-  render() {
-    const { title, message, confirmLabel, cancelLabel, onConfirm } = this.props;
-
-    return (
-      <div className="mailpoet_popup" tabIndex="-1">
-        <div className="mailpoet_popup_wrapper">
-          <button className="mailpoet_modal_close" onClick={() => this.close()} />
-          {title &&
-            <div className="mailpoet_popup_title">
-              <h2>{title}</h2>
-            </div>
-          }
-          <div className="mailpoet_popup_body clearfix">
-            <p className="mailpoet_hp_email_label">{message}</p>
-            <button className="button button-secondary" onClick={() => this.close()}>
-              {cancelLabel}
-            </button>
-            <button className="button button-primary" onClick={() => { onConfirm(); this.close(); }}>
-              {confirmLabel}
-            </button>
-          </div>
+const ConfirmAlert = props => (
+  <div className="mailpoet_popup" tabIndex="-1">
+    <div className="mailpoet_popup_wrapper">
+      <button className="mailpoet_modal_close" onClick={() => props.onClose()} />
```

Comment:
We should use function `onClose` hear that would hide the modal and call the `props.onClose`